.circleci/config.yml

```diff
@@ -39,6 +39,45 @@ jobs:
       - run:
           name: Run UnitTest
           command: ./gradlew test
+   ios_test:
+    macos:
+      xcode: "10.2.0"
+    working_directory: ~/project/example/ios
+    environment:
+      FL_OUTPUT_DIR: output
+    steps:
+      - checkout:
+          path: ~/project
+      - restore_cache:
+          keys: 
+            - flutter-cache
+      - run:
+          name: download flutter SDK
+          command: if ! test -f "flutter_sdk.zip"; then curl -o flutter_sdk.zip https://storage.googleapis.com/flutter_infra/releases/stable/macos/flutter_macos_v1.5.4-hotfix.2-stable.zip; fi
+      - run:
+          name: unzip flutter SDK
+          command: unzip flutter_sdk.zip
+      - run:
+          name: export flutter path
+          command: echo 'export PATH="$PATH:`pwd`/flutter/bin"'  >> $BASH_ENV 
+      - run:
+          name: Fetch CocoaPods Specs
+          command: curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh  | bash -s cf
+      - run: flutter doctor
+      - run: 
+          name: Flutter build
+          command: cd ..; flutter build aot
+      - run:
+          name: Install CocoaPods
+          command: pod install
+      - run:
+          name: Build and run tests
+          command:  xcodebuild   -workspace Runner.xcworkspace   -scheme Runner   -sdk iphonesimulator   -destination 'platform=iOS Simulator,name=iPhone X,OS=12.2'   test
```

Comment:
You can pipe `xcodebuild` to [xcpretty](https://github.com/xcpretty/xcpretty) to get an easier to read build output from Xcode.

xcpretty is already installed on CircleCI.